components/ControllableExpertiseSelector.js

```diff
@@ -0,0 +1,300 @@
+/* globals $, promptError: false */
+import { useState, useEffect } from 'react'
+import IconButton from './IconButton'
+import styles from '../styles/components/ControllableExpertiseSelector.module.scss'
+import { stringToColor } from '../lib/utils'
+import useUser from '../hooks/useUser'
+
+export default function ControllableExpertiseSelector() {
+  const [keyphrases, setKeyphrases] = useState({ active: [], inactive: [] })
+  const [includedKeyphrases, setIncludedKeyphrases] = useState([])
+  const [clickedKeyphrase, setClickedKeyphrase] = useState('')
+  const [recommendedPapers, setRecommendedPapers] = useState([])
+  const [activePapers, setActivePapers] = useState({})
+  const [keyphraseColors, setKeyphraseColors] = useState({})
+  const [paperWeights, setPaperWeights] = useState({})
+  const { user } = useUser()
```

Comment:
I was mistaken about the accessToken. You do need to pass it for requests to the API. You can retrieve it like so:
```suggestion
  const { user, accessToken } = useUser()
```

Reply:
Thanks added.